reactor-core/src/test/java/reactor/core/publisher/FluxPublishOnTest.java

```diff
@@ -1186,8 +1184,15 @@ public void mapNotifiesOnce() throws InterruptedException {
 			                            }
 		                            }));
 
+		final long start = System.currentTimeMillis();
 		for (int i = 0; i < COUNT; i++) {
-			while (s.emitNext(i).hasFailed() );
+			long busyLoops = 0;
+			while (s.tryEmitNext(i).hasFailed() ) {
```

Comment:
```suggestion
			while (s.tryEmitNext(i).hasFailed()) {
```

Reply:
the extra space appears in several other places, I'll fix it